### DISCUSSION TITLE
Is there an option to only click ads on sites I manually enable?

### DISCUSSION THREAD

**Discussion by Tyfyter** [dhowe/AdNauseam#2350] [Q&A] [answered]
I'd like to default to just blocking ads, whether by using this or another adblocker, but I can't make much sense of a substantial amount of the settings, is there a way to do this?

**Comment by dhowe** [comment]
@mneunomne ?

**Comment by mneunomne** [comment, marked as answer]
__AdNauseam currently doesn't have a per-website clicking setting.__

We could implement in a simple way it but it would require to create a full extra page on the dashboard just to add this option. 

Perhaps best scenario would be to create some form of syntax for clicking, something that would allow the user to specifically pick either "_websites-not-to-be-clicked_" or "_websites-to-be-clicked_". Maybe as a dynamic-rule? But that would be a lot of implementation for a specific use case.

This has been previously discussed in various occasions actually: https://github.com/dhowe/AdNauseam/issues/1912 https://github.com/dhowe/AdNauseam/discussions/1831 https://github.com/dhowe/AdNauseam/issues/1966

we can set it to some closer milestone so we can implement in some form in a couple of releases.
